.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "eslint.workingDirectories": [
+        "./app",
+        "./cms"
+    ]
+}
```

Comment:
Oh, I see. Thanks!

Reply:
> Thanks for the PR! But do we need this `.vscode`? This is definitely useful for folks doing the stuff over VS Code (that includes me); however, this may seem to be a not-needed by other IDE users.
> 
> PS. Except for this comment, all other changes are good to be merged. Well done!
> 
> Thank you!

Ohh yeah, that's why I kept it as a separate commit, incase we need to revert😅

Reply:
> > But do we need this .vscode?
> 
> It's ok to commit editor configs like these. Has no negative effect for those not using that specific tool, but quite beneficial for those who do.

Ohh awesome!